ui/jobs/jobs.js

```diff
@@ -2491,6 +2499,105 @@ class Bars {
   }
 
   setupBrd() {
+    // DoT
+    const causticBiteBox = this.addProcBox({
```

Comment:
Do you there should be a straight shot proc box?

Reply:
Seems it will make procs block a little crowded, but not bad to be there.